### DISCUSSION TITLE
"Play Time" Resetting

### DISCUSSION THREAD

**Discussion by PaulNevada** [SteamRE/SteamKit#1248] [General]
### What problem is this feature trying to solve?

For the titles that are non-refundable anymore - to make it possible to start the game from the very beginning...

### How would you like it to be solved?

VIA some "hidden" command...

### Have you considered any alternative solutions

Yes, but reset_all_stats command for the steam client doesn't reset the play time...

**Comment by xPaw** [comment]
That's not a thing.

**Reply by PaulNevada** [reply]
What do you mean?

**Reply by PaulNevada** [reply]
Is it possible to do?

**Comment by zxcvqwerasdf** [comment]
You can't reset playtime because this value is only for increment

**Comment by PaulNevada** [comment]
Can it be bypassed while playing a game?
